ios/FluentUI/Badge Field/BadgeField.swift

```diff
@@ -436,18 +438,20 @@ open class BadgeField: UIView {
         }
     }
 
-    @objc open func heightThatFits(numberOfLines: Int) -> CGFloat {
+    @objc open func heightThatFits(badgeHeight: CGFloat, numberOfLines: Int) -> CGFloat {
         // Vertical margin not applicable to top line
-        let heightForAllLines = CGFloat(numberOfLines) * (Constants.badgeHeight + Constants.badgeMarginVertical)
+        let heightForAllLines = CGFloat(numberOfLines) * (badgeHeight + Constants.badgeMarginVertical)
```

Comment:
Shouldn't the badges also have unlimited number of lines in the scenario where the badge field has unlimited number of lines?

Reply:
Not sure. I've never seen a `BadgeView` with multiple line support, and even if they did, there's still a difference between "let `BadgeView`s flow across multiple lines" and "each `BadgeView` can use as many lines as it wants".

I'm inclined to leave the behavior as-is, but that might be an interesting follow-up for design in the future.